packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,24 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let filterDotFiles = function (x) {
+    return x[0] === ".";
+  };
+  let copyFilesFilter = () => false;
```

Comment:
I feel the naming of the functions could be improved.

What about `copyNoFilter` or `noFilterFn`?